### DISCUSSION TITLE
Doubt with the behaviour of is_checked in chakra.checkbox

### DISCUSSION THREAD

**Discussion by Gelraiser** [reflex-dev/reflex#2926] [Q&A]
Is it normal that if you initialise the is_checked=True property of chakra.checkbox, it gets locked and you cannot change the state of the checkbox by clicking on it?

**Comment by picklelo** [comment]
@Gelraiser yes this is expected, as setting it to True will make it always checked. If you want to bind it to a state, you can do something like `is_checked=State.checked`. Or don't set the prop if you want it to uncontrolled, and simply set the `on_change` to get the value when changed.

**Reply by Gelraiser** [reply]
Thank you very much for the answer. 
What I'm looking for is that when I get the checkbox status data from a database table, the checkbox appears activated or not, depending on the table. That property seemed to me to be the right one, but apparently it is not. I will look for another method.
Although I don't understand what this property behaviour is for.
